src/zenml/zen_stores/migrations/versions/3e86a04263a5_optimizing_database.py

```diff
@@ -0,0 +1,235 @@
+"""optimizing database [3e86a04263a5].
+
+Revision ID: 3e86a04263a5
+Revises: 0.44.2
+Create Date: 2023-09-25 16:59:56.761323
+
+"""
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "3e86a04263a5"
+down_revision = "0.44.2"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    """Upgrade database schema and/or data, creating a new revision."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("pipeline_deployment", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column(
+                "client_version",
+                sqlmodel.sql.sqltypes.AutoString(),
+                nullable=False,
+            )
+        )
+        batch_op.add_column(
+            sa.Column(
+                "server_version",
+                sqlmodel.sql.sqltypes.AutoString(),
+                nullable=False,
+            )
+        )
+        batch_op.alter_column(
+            "pipeline_configuration",
+            existing_type=sa.TEXT(),
+            type_=sa.String(length=16777215).with_variant(
+                mysql.MEDIUMTEXT(), "mysql"
+            ),
+            existing_nullable=False,
+        )
+        batch_op.alter_column(
+            "client_environment",
+            existing_type=sa.TEXT(),
+            type_=sqlmodel.sql.sqltypes.AutoString(),
+            existing_nullable=False,
+        )
```

Comment:
Might cause issues to change this type - probably we changed it to TEXT before because someone reached the limit of regular strings before

Reply:
Must be fixed now.